README.md

```diff
@@ -1,5 +1,9 @@
 # GOV.UK Component Guide
 
+**Component guides now live within applications and are generated using the [govuk_publishing_components gem](https://github.com/alphagov/govuk_publishing_components). This application is being retired.**
+
+**A list of all component guides is available here: https://docs.publishing.service.gov.uk/manual/components.html. Static components are listed here: https://govuk-static.herokuapp.com/component-guide/**
+
```

Comment:
This is good, but I wonder if it's clear enough - I read the preview of this change and immediately found myself skipping past the bit in bold at the top. Would it be worth moving everything below this into a separate file, linked to by something like 'read the original documentation for this application'?

Reply:
Agreed, but do you think we could remove that risk when we move this to the gds-attic?

Reply:
This one-liner was sufficient for specialist-frontend, multipage-frontend and contact-frontend. Which I think is ok given it's going to the attic.